lib/view/components/feed/event_invite.dart

```diff
@@ -67,12 +67,14 @@ class _EventInvitationState extends State<EventInvitation> {
                         children: [
                           Text('${widget.inviterName} invited you to join',
                               style:
-                                  const TextStyle(fontWeight: FontWeight.bold)),
+                                  const TextStyle(fontWeight: FontWeight.bold),
+                              overflow: TextOverflow.ellipsis,),
```

Comment:
I see you tried to handle the overflow error on the event invites, however when I run it on my side I still get an overflow. Will send you a screen shot